services/viva-ms/lambdas/createVivaCase.js

```diff
@@ -110,20 +121,14 @@ async function putRecurringVivaCase(PK, workflowId, period) {
       SK: `${PK}#CASE#${id}`,
       createdAt: timestampNow,
       updatedAt: timestampNow,
-      formId: ssmParams.recurringFormId,
       status: initialStatus,
       provider: CASE_PROVIDER_VIVA,
       details: {
         workflowId,
         period,
       },
-      answers: [],
-      currentPosition: {
-        currentMainStep: 1,
-        currentMainStepIndex: 0,
-        index: 0,
-        level: 0,
-      },
+      activeFormId: initialRecurringForm.id,
+      forms: [initialRecurringForm],
```

Comment:
Was this not supposed to be an object with the id as a key?

Reply:
Hmm...

Reply:
Ska nu vara löst. Kom på ärendet initialt bör innehålla alla relevant formulär. Dvs löpande samt stickprov.
Id'n till formulären sparas lämpligen under ssm params